src/services/class.service.ts

```diff
@@ -58,6 +59,12 @@ export class ClassService {
       const embeddedMessage: discord.MessageEmbed = new discord.MessageEmbed();
 
       embeddedMessage.setColor('#0099ff').setTitle(`${invalidClass} Not Found`);
+      if (Constants.ShouldShowAuthorOnRegister) {
+        embeddedMessage.setAuthor({
+          name: message.member?.displayName ?? '',
+          iconURL: message.author.avatarURL() ?? '',
```

Comment:
might want to consider making a function that returns a `MessageEmbedAuthor` instead of having this everywhere